rakelib/examples_config.yml

```diff
@@ -0,0 +1,16 @@
+example_type_data:
```

Comment:
I figured a yml file would be easiest for people to work with. If we want to add another example type, such as running the app with a new generator option, you just add the name and generator options and you're done. That's it.